### ISSUE TITLE
Issue on excuting cache:clear

### ISSUE THREAD

**Issue by ghost** [phiamo/MopaBootstrapBundle#41]
Hi,
When I excuted app/console cache:clear --env=prod --no-debug ,
I received the following error:

*\* [out :: 192.168.5.72] Clearing the cache for the prod environment with debug true
*\* [out :: 192.168.5.72]
*\* [out :: 192.168.5.72]
*\* [out :: 192.168.5.72]
*\* [out :: 192.168.5.72]   [RuntimeException]
*\* [out :: 192.168.5.72]   Unable to load asset from URL "http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"
*\* [out :: 192.168.5.72]
*\* [out :: 192.168.5.72]
*\* [out :: 192.168.5.72]
*\* [out :: 192.168.5.72] cache:clear [--no-warmup]

The machine I was working with is through proxy to connect internet, and I doubt it's the reason why the above error occurs.
Is it possible to let MopaBootstrapBundle holds the jquery source code instead of fetching them by internet?

Thanks in advanced!
